packages/adminrouter/extra/src/test-harness/tests/test_agent.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) Mesosphere, Inc. See LICENSE file for details.
+
+import grp
+
+import psutil
+import pytest
```

Comment:
2.) I think when we merged AR into the main repo it stopped making sense to check its code quality with separate rules.